modules/expandable-panel/ExpandablePanel.tsx

```diff
@@ -15,6 +15,7 @@ import {
 import { useWindowSize } from "@diana-ui/hooks";
 import { Icon } from "@diana-ui/icon";
 import { BodyHighlight } from "@diana-ui/typography";
+import { ResizeObserver } from "@juggle/resize-observer";
```

Comment:
I think we should do it like this to use the native observer if possible:
```
import { ResizeObserver as Polyfill } from '@juggle/resize-observer';
const ResizeObserver = window.ResizeObserver || Polyfill;
```

 https://github.com/juggle/resize-observer#switching-between-native-and-polyfilled-versions